pkg/kubelet/podresourcesclient.go

```diff
@@ -13,7 +13,7 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
-	"k8s.io/kubelet/pkg/apis/podresources/v1"
+	v1 "k8s.io/kubelet/pkg/apis/podresources/v1"
```

Comment:
nit: shall we set it to a less ambiguous alias while we're at it? `podresourcesv1`?

Reply:
(also applies to other files where this alias was added :wink:)